README.md

```diff
@@ -53,7 +53,63 @@ If you are a bit lost, the following terminology might help to clarify some aspe
 
 **Api** : A wrapper for the database — it allows a user of the emoji bar to store the click count ... somewhere, and then retrieve that amount when desired
 
+# Testing Methodology
+
+## Client
+
+No testing yet...
+
+## API
+
+No testing yet...
+
+## Snippet
+
+Currently only the **snippet** portion has automated testing setup.
+
+Now, the snippet is _not_ a JavaScript module, so the testing offered a challenge. The way it was solved is this case was to have a conditional export of each component that requires testing. You'll notice at the end of the snippet file `snippet/index.js` a series of lines that look like this:
```

Comment:
If possible, write in present-tense, ex:

> Because the javascript snippet is not a module, testing requires conditional export of each component...